main.cpp

```diff
@@ -677,6 +820,8 @@ struct Storage
     bool partitionDrive(char driveLetter, double shrinkSize, int sizeAvailable);
     // store data to drive
     bool storeDataToDrive(std::string fileName, double memorySize, char driveLetter = 'C');
+    // new member function
+    Storage installNewStorage(int threshold, int startingValue);
```

Comment:
a Storage instance will return another Storage?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Storage instance...
- [ ] revise